composer.json

```diff
@@ -10,20 +10,20 @@
     "homepage": "https://github.com/selective-php/basepath",
     "license": "MIT",
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "psr/http-server-middleware": "^1.0"
     },
     "require-dev": {
         "overtrue/phplint": "^1.1",
         "phpstan/phpstan": "*",
-        "phpunit/phpunit": "^7.0",
+        "phpunit/phpunit": "^8.0",
         "squizlabs/php_codesniffer": "^3.4",
         "slim/slim": "^4.3",
         "slim/psr7": "^0.6.0"
     },
     "scripts": {
         "test": "phpunit --configuration phpunit.xml",
-        "test-coverage": "phpunit --configuration phpunit.xml --coverage-clover build/logs/clover.xml --coverage-html build/coverage",
+        "test-coverage": "phpunit --do-not-cache-result --configuration phpunit.xml --coverage-clover build/logs/clover.xml --coverage-html build/coverage",
```

Comment:
It seems that the `PHPUnit 8` will generate cached result.

To avoid above issue, using the `--do-not-cache-result` option to avoid this issue.